action.yml

```diff
@@ -6,6 +6,11 @@ inputs:
       type: string
       default: nrfconnect/sdk-nrf
       description: 'target repository for manifest PR'
+    base-branch:
```

Comment:
If release branch names match between repos --> alternative to use github.base_ref

Reply:
they don't match.

Reply:
if they do not match --> needs input.